apps/arweave/src/ar_peers.erl

```diff
@@ -34,23 +37,47 @@
 %% The maximum number of peers to return from get_peers/0.
 -define(MAX_PEERS, 1000).
 
-%% The number of failed requests in a row we tolerate before dropping the peer.
--define(TOLERATE_FAILURE_COUNT, 20).
+%% Minimum average_success we'll tolerate before dropping a peer.
+-define(MINIMUM_SUCCESS, 0.8).
+-define(THROUGHPUT_ALPHA, 0.05).
```

Comment:
I suggest to comment it is used in the transfer throughput EMA, which is approximately the average over 1 / 0.05 = 20 data points.

Reply:
It turns out its roughly: the last 90 data points contribute 99% of the throughput average. I added a comment explaining the logic here: https://github.com/ArweaveTeam/arweave/pull/443/commits/fbfa3d54ecfc630d5b4534c17cf35d7e3e911aad

Reply:
Perhaps that's too many? Would we prefer to only consider 20 data points?

Reply:
Oops, in my previous explanation I forgot one part of the equation (a datapoint is immediately scaled by Alpha when it is added to the average). With the correct interpretation its roughly the last 31 points that contribute 99% of the average. Updated commit with the comment: https://github.com/ArweaveTeam/arweave/pull/443/commits/8937471296bfe63478c22df5baf505576ffe0654